dyc/top.py

```diff
@@ -0,0 +1,140 @@
+import sys
+import re
+import fileinput
+import copy
+import linecache
+import click
+from .utils import (
+    get_leading_whitespace,
+    BlankFormatter,
+    get_indent,
+    add_start_end,
+    get_file_lines,
+)
+from .base import Builder
+
+
+class TopBuilder(Builder):
+    def __init__(self, filename, config, placeholders=False):
```

Comment:
It looks like this `__init__` method is doing just like the one in `MethodBuilder`. Looks like a fit to the base `Builder.__init__`.

Reply:
#26 Handles this